.github/TEST_MATRIX.md

```diff
@@ -12,6 +12,7 @@ Molecule unit tests
 |![core/display_tuning unit testing](https://github.com/bluebanquise/bluebanquise/workflows/core/display_tuning%20unit%20testing/badge.svg)           | x | x |   |
 |![core/dns_client unit testing](https://github.com/bluebanquise/bluebanquise/workflows/core/dns_client%20unit%20testing/badge.svg)                   | x | x | x |
 |![core/dns_server unit testing](https://github.com/bluebanquise/bluebanquise/workflows/core/dns_server%20unit%20testing/badge.svg)                   | x | x |   |
+|![core/dhcp_server unit testing](https://github.com/bluebanquise/bluebanquise/workflows/core/dhcp_server%20unit%20testing/badge.svg)                 | x |   |   |
```

Comment:
Can you please keep the roles alphabetically sorted?

Reply:
Yes, I will correct it.

Reply:
Corrected in commit: fd4d355